packages/core-svelte/src/components/TextDropdownComponent.svelte

```diff
@@ -344,8 +361,10 @@
 <span id="{id}"
       on:keydown={onKeyDown}
       use:clickOutsideConditional={{enabled: dropdownShown}}
-      on:click_outside={endEditing}
```

Comment:
Are you sure that both click outsides do not clash? Besides, why have two. Please comment.

Reply:
Sure: the first adds a (conditional) listener to the whole document, this listener will create the second (click_outside) event, which is handled in this component.

Reply:
Please add comment to the code

Reply:
Done, added in clickOutside class and click handler